packages/app-form-builder/src/admin/components/FormEditor/Context/useFormEditorFactory.tsx

```diff
@@ -359,7 +359,7 @@ export const useFormEditorFactory = (
                 self.setData(data => {
                     data.steps.push({
                         id: mdbid(),
-                        title: `Step ${data.steps.length + 1}`,
+                        title: `Step`,
```

Comment:
This change was done because we could run into situtation when can have newly created steps with the same name like `Step 3`.
Example of how we could have such situation:
1. User creates 3 steps (order of steps: `Step 1`, `Step 2`, `Step 3`);
2. User changes position of the third step, so it becomes the second one (new order of steps is: `Step 1`, `Step 3`, `Step 2` );
3. User deletes `Step 2`, as we remember it is the last step;
4. User creates new step and the name of that step will be `Step 3` (order of steps: `Step 1`, `Step 3`, `Step 3`);
So I thought that it would be better if remove index for steps except the FIRST ONE STEP, it will always have name `Step 1`.